addons/l10n_ch/report/swissqr_report.xml

```diff
@@ -22,8 +22,6 @@
         <template id="l10n_ch_swissqr_template">
             <t t-set="o" t-value="o.with_context(lang=lang)"/>
             <t t-call="web.external_layout">
-                <!-- add class to body tag -->
```

Comment:
for XPATH safety (in case someone customized the report and used `script` in XPATH, very unlikely but it's better to be safe than sorry) this could be changed to:

```suggestion
                <!-- TODO master: remove this -->
                <script t-if="0">document.body.className += " l10n_ch_qr";</script>
```